src/extension.ts

```diff
@@ -74,24 +75,29 @@ export function activate(ctx: vscode.ExtensionContext): void {
       return;
     }
     var cwd = vscode.workspace.rootPath;
-    fileName = path.relative(cwd, fileName);
+    if(cwd != null)
+      fileName = path.relative(cwd, fileName);  
     var output = vscode.window.createOutputChannel('Runner: ' + action + ' ' + fileName);
-    output.show();
+    output.show(vscode.ViewColumn.Two);
+    editor.show()
     var sh = win32 ? 'cmd' : '/bin/sh';
     var shflag = win32 ? '/c' : '-c';
     var args = document.isDirty || document.isUntitled ? [shflag, action] : [shflag, action + ' ' + fileName];
     var child = cp.spawn(sh, args, { cwd: cwd, detached: false });
+    var clearPreviousOutput = vscode.workspace.getConfiguration('runner')['clearPreviousOutput'] || true;
```

Comment:
as the [default value](https://github.com/mattn/vscode-runner/blob/bce25b04ace32e66b0e919571a68ed1f14ad936e/package.json#L60) is `true` I think `true` is fine